easier68k-cli/example.x68

```diff
@@ -0,0 +1,15 @@
+* Equates section
```

Comment:
I think we should move this under a /easier68k-cli/test directory for the purpose of packaging, so we can exclude the testing files from the production release.